Extensions/Lighting/lightruntimeobject.js

```diff
@@ -1,11 +1,14 @@
 gdjs.LightRuntimeObject = function (runtimeScene, lightObjectData) {
   gdjs.RuntimeObject.call(this, runtimeScene, lightObjectData);
 
-  this._radius = parseFloat(lightObjectData.content.radius);
+  this._radius =
+    typeof lightObjectData.content.radius === 'string'
```

Comment:
unless radius was once stored as a string, you should be able to always do `this._radius = lightObjectData.content.radius`. No need to handle string, they should never appear here. If they do, it's a configuration problem in the extension.